contrib/seccomp/seccomp_default.go

```diff
@@ -408,7 +408,7 @@ func DefaultProfile(sp *specs.Spec) *specs.LinuxSeccomp {
 			Args:   []specs.LinuxSeccompArg{},
 		})
 		fallthrough
-	case "x86":
+	case "386":
```

Comment:
From the Go libseccomp library I don't see any arch "386"; they support "x32" for 32-bit Intel as far as I can tell?

Reply:
Sorry for the noise--just realized you are switching on `runtime.GOARCH` where 386 is correct